assets/css/ionicons.css

```diff
@@ -0,0 +1,1479 @@
+@charset "UTF-8";
```

Comment:
can we use font awesome instead? http://fortawesome.github.io/Font-Awesome/get-started/